Move documentation from pods READMEs and update for new paths (#1656)
ManageIQ/manageiq-documentation: installing_on_kubernetes/_topics/configure_oidc.md

## Configuring OpenID-Connect Authentication

To run with OpenID-Connect Authentication, customize the following example as required to fit your environment.
For this example we tested with Keycloak version 11.0

Create a secret containing the OpenID-Connect's `Client ID` and `Client Secret`.

> The values for `CLIENT_ID` and `CLIENT_SECRET` come from your Keycloak client definition.

Reviewer: ```suggestion
The values for `CLIENT_ID` and `CLIENT_SECRET` come from your authentication provider's client definition.
```

```bash

> $ oc create secret generic <name of your openshift client secret> --from-literal=CLIENT_ID=<your Keycloak client ID> --from-literal=CLIENT_SECRET=<your Keycloak client secret>

Reviewer: ```suggestion
$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your client ID> --from-literal=CLIENT_SECRET=<your client secret>
```

````
Modify the CR with the following values:
```yaml
httpdAuthenticationType: openid-connect

> oidcProviderURL: https://<your keycloak FQDN>/auth/realms/<your Keycloak Realm>/.well-known/openid-configuration

Reviewer: ```suggestion
oidcProviderURL: https://<your auth provider's FQDN>/auth/realms/<your realm>/.well-known/openid-configuration
```

> oidcClientSecret: <name of your openshift client secret>

Reviewer: ```suggestion
oidcClientSecret: <name of your kubernetes secret>
```

## Configuring OpenID-Connect Authentication

To run with OpenID-Connect Authentication, customize the following example as required to fit your environment.
For this example we tested with Keycloak version 11.0

Create a secret containing the OpenID-Connect's `Client ID` and `Client Secret`.
The values for `CLIENT_ID` and `CLIENT_SECRET` come from your authentication provider's client definition.

```bash

> $ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider's client ID> --from-literal=CLIENT_SECRET=<your auth provider's client secret>

Reviewer: Ugh, sorry, I didn't realize the single quote would cause the bash syntax highlight to be annoying - what about on the following:

```suggestion
$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider client ID> --from-literal=CLIENT_SECRET=<your auth provider client secret>
```

or

```suggestion
$ oc create secret generic <name of your kubernetes secret> --from-literal=CLIENT_ID=<your auth provider\'s client ID> --from-literal=CLIENT_SECRET=<your auth provider\'s client secret>
```

Reviewer: This would apply to the other places we did a quote in the placeholder as well.
